src/plugins/synced-lyrics/providers/NetEase.ts

```diff
@@ -0,0 +1,340 @@
+// Code adapted from https://greasyfork.org/en/scripts/548724-youtube-music-spotify-%E7%BD%91%E6%98%93%E4%BA%91%E6%AD%8C%E8%AF%8D%E6%98%BE%E7%A4%BA
+// which is licenced under the MIT licence
+
+import CryptoJS from 'crypto-js';
```

Comment:
I think we can avoid using CryptoJS.

We might be able to achieve this using native APIs (like Web Crypto API or Node's crypto module).